engine/order_manager.go

```diff
@@ -326,7 +344,17 @@ func (m *OrderManager) Modify(mod *order.Modify) (*order.ModifyResponse, error)
 	// Notify observers.
 	if err != nil {
```

Comment:
I think we need to invert this comparison.

Reply:
Thanks for noticing! :)